### ISSUE TITLE
started getting crashes with newest 1.10 versions

### ISSUE THREAD

**Issue by Zygus42** [DoctorWiggles/HelperTools#15]
[crash-2017-02-02_22.02.14-server.txt](https://github.com/DoctorWiggles/HelperTools/files/748140/crash-2017-02-02_22.02.14-server.txt)

Seems to occur when the new skels with masks on are attacking.

ver HelperTools-1.10.2-v4.0b, forge-1.10.2-12.18.3.2185

**Comment by DoctorWiggles** [comment]
Odd, tried a few of those core mods mentioned but couldn't get it to crash. I'll need more info :^)

So It's only happening with one of the new ['stray' skeletons?](http://minecraft.gamepedia.com/Skeleton) 
When exactly does it happen, is it immediately when their AI notices you, or is not until they fire upon you?

Also the crash doesn't seem to mention the skeletons themselves but rather the bomb charm seems to try to spawn it's haunted item entity and hiccup.

For now you can toggle masked skeletons from spawning with the 'extra spooky skeletons' config.
You can also turn off the haunted items with the 'no fun allowed' config.

**Comment by Zygus42** [comment]
Yep, first thing we did was turn off those 2 config options and the problem did not re-occur.  

The problem never happened to me personally, and the people who reported it weren't sure as to exactly what was happening when, nor did they notice if the skeletons were "stray", just that they had the strange masks on and that they then crashed out.

**Comment by DoctorWiggles** [comment]
We'll have to see if the issue pops up for someone else, right now there's too many mods and too little time to test everything there.
Until It can be pinpointed I could be hunting down almost any interaction between any mod, for now I have to assume It's one of the mods in alpha not playing nice, and that it would be fixed later on in their development :^)

**Comment by DoctorWiggles** [comment]
Alright so I've tried to fortify suspected vulnerable areas in the code.
Hopefully it should be [fixed in v4.0d](https://github.com/DoctorWiggles/HelperTools/releases/tag/%281.10%2C2%29-v4.0)
